docs/azure/nRFCloudLocationServices.rst

```diff
@@ -42,3 +42,42 @@ Store the service key into the key vault as follows:
    az keyvault secret set --vault-name ${APP_NAME:-nrfassettracker} \\
      --name nrfCloudCellLocationServiceKey \\
      --file *location of your Cell Location Service Key file*
+
+Assisted GPS Location Service
+*****************************
+
+You can enable your devices to request assisted GPS data using `nRF Cloud's Assisted GPS Location Service API <https://api.nrfcloud.com/v1#tag/Assisted-GPS>`_ in your deployment.
+
+To use the API, set the ``enableNrfCloudAGPSLocationService`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
+
+.. parsed-literal::
+   :class: highlight
+
+    az deployment group create \\
+        --resource-group ${RESOURCE_GROUP:-nrfassettracker} \\
+        --mode Complete \\
+        --name enable-unwiredlabs \\
+        --template-file azuredeploy.json \\
+        --parameters \\
+            appName=${APP_NAME:-nrfassettracker} \\
+            appRegistrationClientId=$APP_REG_CLIENT_ID \\
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
+            keyVaultName=${APP_NAME:-nrfassettracker} \\
+            enableNrfCloudAGPSLocationService=true \\
+            nrfCloudTeamId=*your team ID*
+
+This command enables the ``agpsQueuedDeviceRequestsHandler`` function to resolve A-GPS requests from devices using the nRF Cloud Assisted GPS Location Service API.
+
+Store the service key into the key vault as follows:
+
+.. parsed-literal::
+   :class: highlight
+
+   # Grant the current user set permission to the key vault secrets
```

Comment:
user set or users?

Reply:
It's just one user.

Reply:
This is still a bit unclear to me. Will the user be given a permission to use the key vault secrets or set some permissions?

Reply:
Azure keyvaults have different permission, this command grants them the "set" permission, which is later used in the command:

```
az keyvault secret set ...
                   ^^^
```

Reply:
Okay. It just sounded weird. Set permission it is then.